app/assets/stylesheets/cypress/_test_executions.scss

```diff
@@ -31,3 +33,94 @@
   color: $btn-success-color;
   border-color: $btn-success-border;
 }
+
+.tab-content {
+  margin-top: 1em;
+}
+
+/* * * * * * * * * * * * * * * * * * * * */
+/*   X M L   M A R K U P   V I E W E R   */
+/* * * * * * * * * * * * * * * * * * * * */
+
+.indent {
+  margin-left: 1em;
+}
+
+.markup {
+  /*color: #00008b;*/
+}
+
+.comment {
+  color: $xml-gray;
+}
+
+.text {
+  color: $brand-primary;
+}
+
+.start-tag, .end-tag {
+  /*color: $brand-info;*/
+}
+
+.pi {
+  /*color: #9932CC;*/
+  color: $xml-green;
+}
+
+.attribute-name {
+  color: $xml-orange;
+}
+
+.attribute-value {
+  color: $xml-red;
+}
+
+.navbar {
+  li, .nav-link {
+    display: inline;
+    vertical-align: -webkit-baseline-middle;
+  }
+  
+  .nav-link:hover {
+    background: $gray-lighter;
+    cursor: pointer;
+  }
+}
+
+.navbar-fixed-top {
+  height: 3em;
+  background: $body-bg;
+  border-bottom: 1px solid $panel-default-border;
+
+  .container {
+    padding-left: 3em;
+  }
+}
+
```

Comment:
Most of these selectors need to be more specific or else they'll effect the rest of the site in ways you probably don't intend.